src/pipeline_manager.py

```diff
@@ -76,6 +60,24 @@ def prepare_masks(dev_mode, logger, params):
                       small_annotations_size=params.small_annotations_size)
 
 
+def prepare_metadata(train_data, valid_data, test_data, public_paths, logger, params):
+    logger.info('creating metadata')
+
+    meta = generate_metadata(data_dir=params.data_dir,
+                             meta_dir=params.meta_dir,
+                             masks_overlayed_prefix=params.masks_overlayed_prefix,
+                             competition_stage=params.competition_stage,
+                             process_train_data=train_data,
+                             process_validation_data=valid_data,
+                             process_test_data=test_data,
+                             public_paths=public_paths)
+
+    # metadata_filepath = os.path.join(params.meta_dir, 'stage{}_metadata.csv').format(params.competition_stage)
+    metadata_filepath = os.path.join(params.meta_dir, 'DEBUG_TEST.csv').format(params.competition_stage)
```

Comment:
@jakubczakon I think you forgot to delete this line

Reply:
@apyskir :) thanks